src/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.markdown

```diff
@@ -7,6 +7,8 @@ layout: reference
 
 SAP Concur is pleased to welcome you as a partner. We encourage all Concur App Center partners to create comprehensive marketing plans that incorporate PR, social media, digital marketing, events and sales training to help make the partnership a success. This toolkit provides guidelines and templates for referencing the SAP Concur brand and collaborating on marketing efforts.
 
+**The PDF version of this toolkit is [available here](https://developer.concur.com/manage-apps/go-market-docs/app-center-partner-marketing-toolkit.pdf).**
```

Comment:
I'm getting ready to delete this PDF as redundant so we probably should not link to it.